README.md

```diff
@@ -73,6 +73,29 @@ pre_build_command: "apt-get update -y -q && apt-get install -y -q example"
 
 macOS and Windows platform support will be available soon.
 
+### Welcome a first-time contributor
+
+This is example of a recommended workflow for creating a comment with the
+specified Markdown text when a first-time contributor opens an issue or pull
+request:
```

Comment:
I think it’s worth mentioning that if `pull_request_message` or `issue_message` is omitted, no comment will be posted on PRs / issues.